src/fusiondls/settings.py

```diff
@@ -176,3 +176,57 @@ def __post_init__(self):
         Tcool = np.append(0, Tcool)
         Lalpha = np.fromiter((self.cooling_curve(dT) for dT in Tcool), float)
         self.Lz = [Tcool, Lalpha]
+
+
+def make_SparRange(geometry, mode="equally_spaced_poloidal", num_locations=3):
+    """
+    Create a range of parallel front positions to use in SimulationInputs.
+
+    Note: when using equally_spaced_poloidal, the function finds the closest parallel
+    locations to a list of equally spaced poloidal ones. This can result in duplicates
+    due to resolution constraints which will be deleted, resulting in fewer locations
+    than desired.
+    TODO: fix the above in some way, maybe through refinement?
+
+    Parameters
+    -------
+    profile : MagneticGeometry
+        Magnetic geometry object, either created from an eqdsk file of SOLPS case
+    mode : str
+        Mode to create SparRange. Options are:
+        - "equally_spaced_poloidal": Points are equally spaced in poloidal plane
+        - "equally_spaced_parallel": Points are equally spaced in parallel plane
+        - "target_and_xpoint": Solve only at target and X-point
+        - "target": Solve only at target (i.e. point of detachment threshold)
+    num_locations : int
```

Comment:
Need to add an extra newline to get this to render properly in the docs.

```suggestion
    mode : str
        Mode to create SparRange. Options are:
        
        - "equally_spaced_poloidal": Points are equally spaced in poloidal plane
        - "equally_spaced_parallel": Points are equally spaced in parallel plane
        - "target_and_xpoint": Solve only at target and X-point
        - "target": Solve only at target (i.e. point of detachment threshold)
    num_locations : int
```